### PR TITLE
Tanzu docs

### PR DESCRIPTION
**Pull request by joewhite101**

### CODE REVIEW

File: vuepress/docs/next/tutorials/devops/installation/tanzu/tanzu-install.md [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+
+```
+  entando.requires.filesystem.group.override: "true"
+  entando.ingress.class: "nginx"
+```
+
+6. Find this property in the file `entando.default.routing.suffix:`
+7. Change the value to `<your nginx ip>.nip.io`. For example, `entando.default.routing.suffix: 35.232.231.65.nip.io`
+
+:::note
```
Comment (nshaw) [inline vuepress/docs/next/tutorials/devops/installation/tanzu/tanzu-install.md:126]: This isn't rendering correctly for me so I think the syntax is off.  I think it should be "::: tip"

File: vuepress/docs/next/tutorials/devops/installation/tanzu/tanzu-install.md [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+
+
+8. Deploy the operator configuration
+
+```
+kubect apply -f sample-configmaps/entando-operator-config.yaml -n entando
+```
+
+9. Open values.yaml in the entando-helm-quickstart
+10. Changed the dbms from `embedded` to `postgresql`
```
Comment (nshaw) [inline vuepress/docs/next/tutorials/devops/installation/tanzu/tanzu-install.md:139]: "Change"

File: vuepress/docs/next/tutorials/devops/installation/tanzu/tanzu-install.md [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+:::
+
+
+8. Deploy the operator configuration
+
+```
+kubect apply -f sample-configmaps/entando-operator-config.yaml -n entando
+```
+
+9. Open values.yaml in the entando-helm-quickstart
```
Comment (nshaw) [inline vuepress/docs/next/tutorials/devops/installation/tanzu/tanzu-install.md:138]: "Open the"

File: vuepress/docs/next/tutorials/devops/installation/tanzu/tanzu-install.md [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+10. Changed the dbms from `embedded` to `postgresql`
+11. Deploy your Entando application
+
+```
+helm template --name=quickstart ./ | kubectl apply -n entando -f -
+```
+
+12. Watch the deployment for completion
+```
+watch kubectl get pods -n entando
```
Comment (nshaw) [inline vuepress/docs/next/tutorials/devops/installation/tanzu/tanzu-install.md:148]: The watch cmd isn't available on Windows Git Bash so I think we used the --watch option in other tutorials.

**Review by nshaw — APPROVED** [review]
A few minor suggestions.

File: vuepress/docs/next/tutorials/devops/installation/tanzu/tanzu-install.md (hunk 1 above)
Comment (joewhite101) [reply vuepress/docs/next/tutorials/devops/installation/tanzu/tanzu-install.md:126]: fixed

File: vuepress/docs/next/tutorials/devops/installation/tanzu/tanzu-install.md (hunk 2 above)
Comment (joewhite101) [reply vuepress/docs/next/tutorials/devops/installation/tanzu/tanzu-install.md:139]: fixed

File: vuepress/docs/next/tutorials/devops/installation/tanzu/tanzu-install.md (hunk 4 above)
Comment (joewhite101) [reply vuepress/docs/next/tutorials/devops/installation/tanzu/tanzu-install.md:148]: fixed

**Pull request merged**